kernel_gateway/gatewayapp.py

```diff
@@ -505,6 +515,7 @@ def init_http_server(self):
         """
         ssl_options = self._build_ssl_options()
         self.http_server = httpserver.HTTPServer(self.web_app,
+                                                 xheaders=bool(strtobool(self.x_headers)),
```

Comment:
Per above, I think we can eliminate the casts (and import) by using the traitlets - as Notebook did.